bm-persona/Dining/DiningDetailViewController.swift

```diff
@@ -0,0 +1,496 @@
+//
+//  DiningDetailViewController.swift
+//  bm-persona
+//
+//  Created by Shawn Huang on 4/4/20.
+//  Copyright © 2020 RJ Pimentel. All rights reserved.
+//
+
+import UIKit
+import CoreLocation
+
+fileprivate let kHeaderFont: UIFont = Font.bold(24)
+fileprivate let kCardPadding: UIEdgeInsets = UIEdgeInsets(top: 16, left: 16, bottom: 16, right: 16)
+fileprivate let kViewMargin: CGFloat = 16
+class DiningDetailViewController: UIViewController {
+    
+    var diningHall: DiningHall?
```

Comment:
What's the reason for making this an optional vs. an implicitly unwrapped optional? It seems like you're force-unwrapping `diningHall` whenever you use it, so there doesn't seem to be a purpose for making this an optional.